eng/common/tools.ps1

```diff
@@ -374,7 +374,7 @@ function InitializeVisualStudioMSBuild([bool]$install, [object]$vsRequirements =
   }
 
   $msbuildVersionDir = if ([int]$vsMajorVersion -lt 16) { "$vsMajorVersion.0" } else { "Current" }
-  return $global:_MSBuildExe = Join-Path $vsInstallDir "MSBuild\$msbuildVersionDir\Bin\msbuild.exe"
+  return $global:_MSBuildExe = Join-Path $vsInstallDir "MSBuild\$msbuildVersionDir\Bin\amd64\msbuild.exe"
```

Comment:
Curious: How does this make sense in our release/3.1 branch❔

Reply:
It doesn't. See https://github.com/dotnet/aspnetcore/pull/32618.

Reply:
Never mind. Thought this was in one of our scripts for a minute.